__tests__/app.test.js

```diff
@@ -376,4 +376,68 @@ describe("get /api/users", () => {
     });
 });
 
+describe("get /api/articles/?sortby=&order=&topic=", () => {
+    test("return an array of article objects sorted by date descending by default", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles).toHaveLength(13);
+                expect(body.articles).toBeSortedBy("created_at", {descending: true})
+
+            });
+    });
+    test("return an array of article objects sorted by author ascending alphabetically", () => {
+        return request(app)
+            .get("/api/articles/?sort_by=author&order=asc")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles).toBeSortedBy('author', { ascending: true })
+            });
+    });
+    test("return an array of article objects sorted by votes with descending default value", () => {
+        return request(app)
+            .get("/api/articles/?sort_by=votes")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles).toBeSortedBy('votes', { descending: true })
+            });
+    });
+    test("return an array of article objects filtered by the chosen topic", () => {
+        return request(app)
+            .get("/api/articles/?topic=cats")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.articles).toHaveLength(1);
+                body.articles.forEach(article => {
+                    expect(article.topic).toBe("cats");
+                });
+            });
+    });
```

Comment:
Please add another test for:
Status 200, valid topic query, but has no articles responds with an empty array of articles, e.g. **?topic=paper.**